app-shells/zsh/patches/zsh-5.4.1.patchset

```diff
@@ -127,4 +125,95 @@ index d1b98ac..eba94d8 100644
  	j->procs->ti.ru_maxrss / 1024 > reportmemory)
  	return 1;
 -- 
-2.11.0
+2.14.1
+
+
+From 882f59c478ca7f8f1367a85c3132fef9bae19151 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Mon, 14 Aug 2017 22:07:38 +0200
+Subject: Dynamic loadable modules
+
+
+diff --git a/configure.ac b/configure.ac
+index bad78f9..36d90c5 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -695,7 +695,6 @@ AC_CHECK_HEADERS(sys/time.h sys/times.h sys/select.h termcap.h termio.h \
+ 		 ncurses/ncurses.h)
+ if test x$dynamic = xyes; then
+   AC_CHECK_HEADERS(dlfcn.h)
+-  AC_CHECK_HEADERS(dl.h)
```

Comment:
What's the problem here? It should normally check the header and not find it, and continue.